python/gui/maas_experiment/settings.py

```diff
@@ -174,7 +125,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = os.environ.get("MAAS_TIMEZONE", 'UTC')
+TIME_ZONE = CURRENT_TIMEZONE
```

Comment:
Does `django` magically import `CURRENT_TIMEZONE` from somewhere else (im still new to django settings)? I don't see `CURRENT_TIMEZONE` defined or imported elsewhere.

Reply:
`CURRENT_TIMEZONE` is coming in from `application_values`. It's defined there because you have to spin up another server instance to use the stuff in `settings.py`, which make cross-process stuff a problem. It's good to have everything in one, safe place.

Reply:
Duh, my bad. Didnt notice the wildcard import at the top! Sorry!